docs/cudf/source/developer_guide/errors.md

```diff
@@ -0,0 +1,27 @@
+# cuDF Error Handling Documentation
+
+This document discusses what to raise given invalid user inputs and how to handle libcudf exceptions in cuDF.
+
+## Overview
+
+cuDF uses [builtin error types](https://docs.python.org/3/library/exceptions.html) to indicate user error.
```

Comment:
Is this intended to imply that cuDF should not use custom exception types?

Reply:
I don't think that's intended. How about
```python
cuDF uses [builtin error types](https://docs.python.org/3/library/exceptions.html) and custom error types to indicate user error.
```

Reply:
I don't think this sentence really adds anything. I would just remove it.